.github/pull_request_template.md

```diff
@@ -1,10 +1,18 @@
-Checklist for reviewer:
+## Description
 
-- [ ] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title).
-- [ ] If the PR comes from a fork, trigger integration CI tests by running the [Push to upstream workflow](https://github.com/mozilla/bigquery-etl/actions/workflows/push-to-upstream.yml) and provide the `<username>:<branch>` of the fork as parameter. The parameter will also show up
-in the logs of the `manual-trigger-required-for-fork` CI task together with more detailed instructions.
-- [ ] If adding a new field to a query, ensure that the schema and dependent downstream schemas have been updated.
-- [ ] When adding a new derived dataset, ensure that data is not available already (fully or partially) and recommend extending an existing dataset in favor of creating new ones. Data can be available in the [bigquery-etl repository](https://github.com/mozilla/bigquery-etl), [looker-hub](https://github.com/mozilla/looker-hub) or in [looker-spoke-default](https://github.com/mozilla/looker-spoke-default/tree/e1315853507fc1ac6e78d252d53dc8df5f5f322b).
+<!--
+Please do not leave this blank
+This PR [adds/removes/fixes/replaces] the [feature/bug/etc].
+-->
 
-For modifications to schemas in restricted namespaces (see [`CODEOWNERS`](https://github.com/mozilla/bigquery-etl/blob/main/CODEOWNERS)):
-- [ ] Follow the [change control procedure](https://docs.google.com/document/d/1TTJi4ht7NuzX6BPG_KTr6omaZg70cEpxe9xlpfnHj9k/edit#heading=h.ttegrcfy18ck)
+## Related Tickets & Documents
+* DENG-XXXX
+* DSRE-XXXX
+
+<!--
+Please reference related Jira tickets, GitHub issues or Bugzilla. This repo has been
+configured to automatically insert hyperlinks for DSRE and DENG tickets.
+See https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/configuring-autolinks-to-reference-external-resources
+-->
+
+Reviewer, please follow [this checklist](https://github.com/mozilla/bigquery-etl/.github/reviewer_checklist.md)
```

Comment:
I expect that putting this in a separate file will make it even more ignored so maybe we're better off not having it to not set expectations we won't meet.  I'm ok with leaving this line here since it doesn't take up much space but I think it's worth considering keeping the checklist and making it a lot shorter.

We could also have the checklist in a separate automated comment like https://github.com/marketplace/actions/github-script#comment-on-an-issue so the preview is cleaner but I don't know if that's better or worse.

Reply:
I agree that there's even less of a chance reviewers are going to look at the checklist now. 
At the same time I think there's value in that list, and I couldn't think of anything there that I would remove without feeling like I removed something sort of important (except for the first item, which is covered in the new template), especially in the case of new reviewers. 
Couple that with the intention of adding the least amount of noise, I figured that one line will work as documentation: very few will actually read it, but we can point folks to it and it gives new hires some direction. Btw I think I will make it **bold** to be purposefully a little more annoying.